src/subcommands/show_syntax_themes.rs

```diff
@@ -82,21 +81,18 @@ index f38589a..0f1bb83 100644
     opt.computed.is_light_mode = is_light_mode;
     let mut config = config::Config::from(opt);
     let title_style = ansi_term::Style::new().bold();
-    let assets = HighlightingAssets::new();
+    let assets = bat::assets::HighlightingAssets::from_binary();
 
     for syntax_theme in assets
-        .theme_set
-        .themes
-        .iter()
-        .filter(|(t, _)| is_light_syntax_theme(t) == is_light_mode)
-        .map(|(t, _)| t)
+        .themes()
+        .filter(|t| is_light_syntax_theme(t) == is_light_mode)
     {
         writeln!(
             writer,
             "\n\nSyntax theme: {}\n",
             title_style.paint(syntax_theme)
         )?;
-        config.syntax_theme = Some(assets.theme_set.themes[syntax_theme.as_str()].clone());
+        config.syntax_theme = Some(assets.get_theme_set().get(syntax_theme).unwrap().clone());
```

Comment:
Usage of currently private `get_theme_set()` here (using `LazyThemeSet.get(theme_name)`)